vite.config.ts

```diff
@@ -17,7 +22,12 @@ export default defineConfig( {
 		rollupOptions: {
 			external: [ '@vue/compat', 'vue', 'vuex' ],
 		},
-	},
+	};
+}
+
+// https://vitejs.dev/config/
+export default defineConfig( {
+	build: getBuildConfig( !!process.env.BUILD_AS_APP ),
```

Comment:
Optional: maybe we should `import { env } from 'process';`?

Reply:
I think I prefer `process.env` as it is such an established pattern and it makes it obvious right away that it is coming from the environment variables directly, without needing to scroll up and see where `env` comes from.

Reply:
We could still have that with `import process from 'process'` if I’m not mistaken – to me it feels weird how Node has some of these modules that you can `import`/`require` but also they just exist by default for some reason. ([The docs](https://nodejs.org/api/process.html#process) say that the explicit import is recommended, apparently.)